application/core/db/MysqlSchema.php

```diff
@@ -18,6 +18,10 @@ public function createTable($table, $columns, $options = null)
             $options = 'DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci';
         }
 
+        if (Yii::app()->getConfig('forceMyISAM') === true) {
```

Comment:
Why not
````
If (App()->db->driverName == 'mysql') {
     $options .= ' ENGINE='.App()->getConfig('mysqlEngine');
}
````

Or better : 
````
        if (empty($options)) {
            $options = 'ENGINE='.App()->getConfig('mysqlEngine').' DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci';
        }
````
in : https://github.com/LimeSurvey/LimeSurvey/blob/d905cdce4ad13e401a2ceb4ac50b13144b799431/application/core/db/MysqlSchema.php#L18

Then : we have the same behaviour in 3.X (and before) and 4.X

Reply:
Good point. This allows for better backwards compatibility. Thanks, I updated the PR.